src/main/resources/configCivcraft.yml

```diff
@@ -5652,7 +5652,7 @@ recipes:
     name: Extract Aether
     type: PYLON
     production_time: 4h
-    fuel_consumption_intervall: 6m
+    fuel_consumption_intervall: 80m
```

Comment:
Maybe a week is too much? When I brought it up with ttk he said he was fine with it, so I made it a week. Honestly it needs to be longer then 12 hours though, maybe something like 2 days?